src/platform_impl/windows/window.rs

```diff
@@ -825,7 +830,9 @@ unsafe fn init<T: 'static>(
             return Err(os_error!(io::Error::last_os_error()));
         }
 
-        WindowWrapper(handle)
+        let hinstance = libloaderapi::GetModuleHandleW(std::ptr::null() as *const _);
```

Comment:
Strictly speaking, it's possible to change a window's HINSTANCE via [`SetWindowLong`](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setwindowlonga), so we should probably use `GetWindowLong` to prevent bugs if somebody (future Winit developers or downstream users) decides to do that.

Reply:
I implemented it using GetWindowLong now